source/usage-examples/findOne.txt

```diff
@@ -0,0 +1,51 @@
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+You can retrieve a single document in a collection by chaining together
+the ``find()`` and ``first()`` methods on a ``MongoCollection`` object.
+You can pass a query filter to the ``find()`` method to query for and
+return documents that match the filter in the collection. If you do not
+include a filter, MongoDB returns all the documents in the collection. The
+``first()`` method returns the first document from the matches.
+
+For more information on querying MongoDB with the Java driver, see our
+guide (TODO: link) on querying documents.
+
+You can also chain other methods to the ``find()`` method
+such as ``sort()`` (TODO: link) which organizes the matched documents in a
```

Comment:
I'm not sure just how much detail we need to go into here, but it might be useful to point out if these other methods are handled in the database or the driver itself (that is, if we're adjusting fields/sorting documents returned by the initial unmodified find() call, or if these calls actually adjust the query itself before it hits the database.

Reply:
I'm not sure either -- seems like where it gets executed would belong in the fundamentals guide, but not sure if it's worth repeating here since it's not central to understanding how to code and modify the example.

Reply:
I can definitely agree with that!